setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = s2geometry
+version = 0.10.0
```

Comment:
What's the allowed format of version? Can this be 0.11.dev or prerelease or something?

0.10 has already been released, so it's probably not great if we end up with loads of variants built at different times claiming to be 0.10.0.